flutter_vlc_player/lib/src/vlc_player_controller.dart

```diff
@@ -97,6 +97,15 @@ class VlcPlayerController extends ValueNotifier<VlcPlayerValue> {
   /// Initialize vlc player when the platform is ready automatically
   final bool autoInitialize;
 
+  /// This is a callback that will be executed once the platform view has been initialized.
+  /// If you want the media to play as soon as the platform view has initialized, you could just call
+  /// [VlcPlayerController.play] in this callback. (see the example)
+  final VoidCallback? _onInit;
```

Comment:
Please, mark the constructor parameters for these fields as `deprecated` since it seems like we should remove them in the next version. Also, I believe it would be great to leave a comment explaining what the user should use instead of them. I think a message similar to the following would be great:
```
This parameter is deprecated, please, use the [addOnInitListener] method instead.
```
Also, I think it would be great to update our example to avoid using deprecated parameters.

Reply:
fixed